### ISSUE TITLE
Is there a way to adjust the height of a UDRow or element?

### ISSUE THREAD

**Issue by murrahjm** [ironmansoftware/universal-dashboard#364]
I have a page that has a few line charts on them.  They're historical charts so I want them to be as wide as they can be, but it would be ideal if I could make them much shorter.  At full size a single chart takes up the whole screen.  If I could configure the height at either the row or chart level that would help a lot.

**Comment by adamdriscoll** [comment]
There isn't a height parameter for UDRow but you could use a UDElement and add some attributes. 

Try this:
```powershell
New-UDElement -Tag div -Attributes @{ className = 'row'; height = '100px' } -Content {
      # Chart goes here
}
```

**Comment by murrahjm** [comment]
That didn't seem to have any effect.  Is this how I should use that?

```powershell
        New-UDElement -Tag div -Attributes @{ className = 'row'; height = '10000px' } -Content {
            # Chart goes here
                                        
            New-UDChart -Title "File Count History - $folder" -Type Line -Endpoint {
                $chartdata | Out-UDChartData -LabelProperty "TimeStamp" -DataProperty "FileCount"
            }
        }
```

**Comment by adamdriscoll** [comment]
@murrahjm Ack. That's me trying to write code without testing it. Try this:

```powershell
$d = New-UDDashboard -Title "Hey" -Content {
    New-UDElement -Tag 'div' -Attributes @{
        className = "row";
        style = @{
            height = "500px";
            backgroundColor = "blue"
        }
    } 
}

Start-UDDashboard -Dashboard $d -Port 10001
```

**Comment by murrahjm** [comment]
ok, so this is interesting.  Your code worked, and created a row of that size.  But when I put a chart object in there it just expanded it to the chart size, which appears to be 1000px.  In fact I could actually see the bright  blue background for a second before the chart loaded.  Here's the code I used for the chart addition.

```PowerShell
$d = New-UDDashboard -Title "Hey" -Content {
    New-UDElement -Tag 'div' -Attributes @{
        className = "row";
        style = @{
            height = "1200px";
            backgroundColor = "blue"
        }
    } -Content {
        $data  = [pscustomobject]@{x=@('a','b','c');y=@(0,1,2)}
        New-UDChart -Labels 'x' -Endpoint {$data | Out-UDChartData -DataProperty y -LabelProperty x}
        
    } 
}

Start-UDDashboard -Dashboard $d -Port 10001 -AutoReload
```

Would I need to do a udelement instead of udchart and use `classname=chart` or something like that?

**Comment by adamdriscoll** [comment]
@murrahjm I'll have to play with this more. You won't be able to replace a chart with UDElement.

**Comment by adamdriscoll** [comment]
Fixed with PR#397
